.buildkite/pipeline-resource-definitions/kibana-serverless-release.yml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-serverless-release
+  description: Initiate kibana serverless releases
+spec:
+  type: buildkite-pipeline
+  owner: 'group:kibana-operations'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / serverless / release
+      description: Initiate kibana serverless releases
+    spec:
+      env:
+        SLACK_NOTIFICATIONS_CHANNEL: '#kibana-operations-alerts'
+        SLACK_NOTIFICATIONS_ENABLED: 'false'
```

Comment:
These may need to be updated - sending you a docs link.

Reply:
```suggestion
        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'false'
```